ProjectWithUMLDiagram/src/com/beyond/person/core/exceptions/InvalidAgeException.java

```diff
@@ -0,0 +1,7 @@
+package com.beyond.person.core.exceptions;
+
+public class InvalidAgeException extends InvalidException{
```

Comment:
@AraChilingaryan exceptions should be specific for age . 
You can have BaseException instead of InvalidException 
And have ValidationException which will extends BaseException
And everything related to validation will throw this exception with corresponding message

Reply:
done